Graphs/Dijikstra.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+This program is for dijkestra
+It finds the shortest path from 1 to n (can be easily changed)
+It also returns the path
+*/
+
+#include <iostream>
+#include <queue>
+#define INF 1e17
```

Comment:
It's better to change it to `const long long INF = (long long)1e17`. For the time being it's considered as a `double`.